
---
float64 pos_x
float64 pos_y
float64 pos_z
float64 zyx_angle_z
float64 zyx_angle_y
float64 zyx_angle_x